tests/pytest/test.py

```diff
@@ -1487,6 +1488,48 @@ def test_promote_u64_to_f64(env):
     r.assertEqual(val, float(2 * i64max + 3))            # u64 + u64 promotes to f64. as prior, not breaking
     r.expect('JSON.TYPE', 'num', '$').equal(['number'])  # promoted
 
+
+def test_mset_replication_in_aof(env):
+    if not env.useAof:
+        env.skip()
```

Comment:
```suggestion
def test_mset_replication_in_aof():
    env = Env(useAof=True)
```
We probably want to create the env here with aof enabled